src/dapp/README.md

```diff
@@ -314,6 +314,7 @@ variables](../hevm/README.md#environment-variables).
 | `DAPP_REMAPPINGS`          | `$(dapp remappings)`       | [Solidity remappings](https://docs.soliditylang.org/en/latest/using-the-compiler.html#path-remapping)                                              |
 | `DAPP_BUILD_OPTIMIZE`      | `0`                        | Activate Solidity optimizer (`0` or `1`)                                                                                                           |
 | `DAPP_BUILD_OPTIMIZE_RUNS` | `200`                      | Set the optimizer runs                                                                                                                             |
+| `DAPP_VIA_IR`              | `0`                        | Change compilation pipeline to go through the Yul intermediate representation (`0` or `1`)                                                                                                           |
```

Comment:
I think the default should be `1` here?

Reply:
The default behavior is unchanged by this PR, it's to use the legacy pipeline, which can be expressed as `DAPP_VIA_IR=0`. We don't want to suddenly switch all the users to using Yul IR, because it's not a drop-in replacement.

Reply:
but isn't the default set to `1` here: https://github.com/dapphub/dapptools/pull/943/files#diff-b2869294406b9b133b905fc0cb95a92a490154bbc16a1de23bc4bc91692973e4R144 ?